util/RestSpecRunner.php

```diff
@@ -26,6 +26,21 @@
 echo "Update elasticsearch submodule\n";
 $git->fetchAll(array('verbose' => true));
 
-$hash = $_SERVER['TEST_BUILD_REF'];
+$out = [];
+exec('./elasticsearch-*/bin/elasticsearch --version', $out);
```

Comment:
It returns this: `Version: 6.0.0-rc1-SNAPSHOT, Build: 6f57a84/2017-09-01T18:07:02.565Z, JVM: 1.8.0_144`